documentation/development-docs/enable-subscription-id-overriding.md

```diff
@@ -0,0 +1,12 @@
+# How to: Enable Subscription ID Overriding in Your Module
```

Comment:
I'm planning to write a guide for service team about how to onborad your module to this feature.
Let me know if you think this is a good idea or not

Reply:
You may add code snippet as the example

Reply:
Good idea. This is just a draft of the document. I'll add more to it in a seprate PR